test/reducers/reducerTest.js

```diff
@@ -243,17 +194,6 @@ describe('reducers', () => {
                    'selected task should not have snoozedAt')
       assert.equal(dateStr, actual[1].snoozedAt,
                    'deselected task should have snoozedAt')
-
-      const snoozedTaskKeys = storage.get('snoozed')
-      assert(snoozedTaskKeys.indexOf('issue-97') < 0,
```

Comment:
Likewise, this is going away, but for the test name "restores snoozed, selected task" to be honest, we need to confirm that the task loses its snoozed status after restoring it. Maybe assert that after TASKS_RESTORE, snoozedAt is null again?